### PR TITLE
x11-wm/qtile: depend on pywlroots-0.15.x versions

### PR DESCRIPTION
**Pull request by mrtnvgr**
Closes: https://bugs.gentoo.org/895722

### CODE REVIEW

**Comment by ftpd** [comment]
@mrtnvgr Why `qtile-0.21.0-r1.ebuild` (current stable!) and `qtile-0.22.1.ebuild` are not included?

**Comment by mrtnvgr** [comment]
> @mrtnvgr Why `qtile-0.21.0-r1.ebuild` (current stable!) and `qtile-0.22.1.ebuild` are not included?

These ebuilds don't have a `pywlroots` as a dependency

**Comment by ftpd** [comment]
> > @mrtnvgr Why `qtile-0.21.0-r1.ebuild` (current stable!) and `qtile-0.22.1.ebuild` are not included?
> 
> These ebuilds don't have a `pywlroots` as a dependency

Right, sorry!

**Comment by mrtnvgr** [comment]
> ## Pull request CI report
> _Report generated at_: 2023-02-23 11:18 UTC _Newest commit scanned_: [4c6660e](https://github.com/gentoo/gentoo/commit/4c6660e7b7e94c90b41e7b2a5949ca3fcf21ff36) _Status_: ❌ **broken**
> 
> New issues caused by PR: https://qa-reports.gentoo.org/output/gentoo-ci/5e1c9889d6/output.html#x11-wm/qtile
> 
> There are existing issues already. Please look into the report to make sure none of them affect the packages in question: https://qa-reports.gentoo.org/output/gentoo-ci/5e1c9889d6/output.html


> "=" operator used without package revision: BDEPEND="=dev-python/pywlroots-0.15.24[python_targets_python3_10(-)?,python_targets_python3_11(-)?,python_targets_python3_9(-)?]"

I can clearly see that package revision is specified, what is wrong here?

**Comment by ceamac** [comment]
https://projects.gentoo.org/qa/policy-guide/dependencies.html#pg0002

Either add `-r0` or use `~` instead of `=`

**Comment by thesamesam** [comment]
We probably want =...-0.15*

**Comment by ceamac** [comment]
Also looking at the bug you may want to use 0.15* instead

File: x11-wm/qtile/qtile-0.22.1-r1.ebuild [hunk 1]
```diff
@@ -37,7 +37,7 @@ RDEPEND="
 		media-sound/pulseaudio
 	)
 	wayland? (
-		dev-python/pywlroots[${PYTHON_USEDEP}]
+		~dev-python/pywlroots-0.15*[${PYTHON_USEDEP}]
```
Comment (ceamac) [inline x11-wm/qtile/qtile-0.22.1-r1.ebuild:40]:
```suggestion
		=dev-python/pywlroots-0.15*[${PYTHON_USEDEP}]
```
`~` is only for non-wildcard exact version

**Comment by thesamesam** [comment]
Thanks!

Some notes:
- I've squashed the commits;
- Added a comment in the ebuild about the pinned dep so we can easily check its status and others know why it's there
- Try to avoid closing bugs until the PR fixing them is merged (no big deal here, just saying for future)

Appreciate the quick fix!

**Comment by thesamesam** [comment]
Merged in dff5e674d0b9e0875bb80773178e4b70738dbcb3, sorry, I lost the `Closes:` tag on squashing :)

**Pull request closed without merging**